src/lib/y2storage/package_handler.rb

```diff
@@ -73,7 +73,22 @@ def add_packages(pkg_list)
     #
     def add_feature_packages(devicegraph)
       used_features = UsedStorageFeatures.new(devicegraph)
-      add_packages(used_features.feature_packages)
+      packages = used_features.feature_packages
+      packages = packages.delete_if { |pkg| unavailable_optional_package?(pkg) }
+      add_packages(packages)
+    end
+
+    # Check if a package is an optional package that is unavailable.
+    # See also bsc#1039830
+    #
+    # @param package [String] package name
+    # @return [Boolean] true if package is optional and unavailable,
+    #                   false if not optional or if available.
+    def unavailable_optional_package?(package)
+      return false unless UsedStorageFeatures::OPTIONAL_PACKAGES.include?(package)
```

Comment:
NP: To fully honor the Uniform Access Principle, this should be a class method, instead of accessing directly to the constant from outside of `UsedStorageFeatures`